get_configs.py

```diff
@@ -17,7 +17,7 @@ def get_config(config: str):
     location = f"config_files/{config}_config.py"
 
     try:
-        if current_dir.endswith("importer"):
+        if "importer" in current_dir.lower():
```

Comment:
changed this to be more general, but having this hardcoded doesn't seem ideal still, want this to be able to find config files whether it is running from the top level directory or a subdirectory, e.g if "importer" in name of current directory import config from top level directory.